src/main/java/uk/gov/hmcts/befta/dse/ccd/definition/converter/JsonTransformer.java

```diff
@@ -40,7 +40,7 @@ public class JsonTransformer {
             "CaseEventToFields", "CaseField", "CaseRoles", "CaseTypeTab" ,"SearchInputFields", "SearchResultFields", "State",
             "WorkBasketInputFields", "WorkBasketResultFields", "Category", "Banner", "CaseType", "ComplexTypes", "EventToComplexTypes",
             "FixedLists", "Jurisdiction", "UserProfile","SearchAlias", "SearchCasesResultFields","Categories",
-            "NoticeOfChangeConfig", "ChallengeQuestion", "RoleToAccessProfiles", "SearchParty", "SearchCriteria");
+            "NoticeOfChangeConfig", "ChallengeQuestion", "RoleToAccessProfiles", "SearchParty", "SearchCriteria","AccessTypeRoles");
```

Comment:
sorry to be super pendantic - can you add a space after the comma here to be consistent!